src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/ItemFilter.java

```diff
@@ -200,7 +200,7 @@ public boolean test(@Nonnull ItemStack item) {
              * and thus only perform .getItemMeta() once
              */
             for (ItemStackWrapper stack : items) {
-                if (SlimefunUtils.isItemSimilar(subject, stack, checkLore, false)) {
+                if (SlimefunUtils.compareItem(subject, stack)) {
```

Comment:
The "checkLore" option is ignored here

Reply:
We already check lore in the method. We don’t specifically need to check for it with a boolean

Reply:
The option exists in the gui yet it has no effect on the check.